lib/rubycritic/generators/text/templates/analysed_module_console_report.erb

```diff
@@ -0,0 +1,12 @@
+<%= name.send(rating.color) %>:
```

Comment:
let's avoid `send` and use the `colorize` method to be more explicit about the coloring usage, wdyt?